.github/workflows/rust.yml

```diff
@@ -11,17 +11,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout sources
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Install stable toolchain
-        uses: actions-rs/toolchain@v1
+        uses: actions-rs/toolchain@v2
```

Comment:
@5Dev24 getting an error here. i think v1 is the maximum available version. can you revert the action-rs references to v1?

>  Unable to resolve action `actions-rs/toolchain@v2`, unable to find version `v2`

Reply:
Downgraded